src/redux/actions/auth.js

```diff
@@ -28,10 +39,10 @@ const signOut = () => async (dispatch, getState, { getFirebase }) => {
   }
 }
 
-const signUp = ({ email, password }) => async (
-  dispatch,
-  getState,
-  { getFirebase }
+const signUp = ({ email, password }: SignInUpArgsType) => async (
+  dispatch: actionTypes,
+  getState: actionTypes,
+  { getFirebase }: actionTypes
```

Comment:
These types are defined under `constants/redux` and `constants/firebase`

Reply:
[aaron/feature/profile-scene 4ca843c] :ok_hand: [profile-scene] use types defined under constants/redux & constants/firebase